src/BundleAdjustmentNLSFunctions.jl

```diff
@@ -267,20 +265,7 @@ function NLPModels.jac_coord!(nls::BundleAdjustmentModel, x::AbstractVector, val
 end
 
 function NLPModels.jac_op_residual(nls::BundleAdjustmentModel, x::AbstractVector)
-  jac_structure!(nls, nls.rows, nls.cols)
-  jac_coord!(nls, x, nls.vals)
-  Jx = jac_op_residual!(nls, nls.rows, nls.cols, nls.vals, nls.Jv, nls.Jtv)
-  return Jx
-end
-
-"""
-    jac_op_residual_update(nls :: BundleAdjustmentModel, x :: AbstractVector)
-
-Update the jacobian operator of the residual wihtout jac_structure! instead
-of using jac_op_residual
-"""
-function jac_op_residual_update(nls::BundleAdjustmentModel, x::AbstractVector)
-  jac_coord!(nls, x, nls.vals)
+  jac_coord_residual!(nls, x, nls.vals)
   Jx = jac_op_residual!(nls, nls.rows, nls.cols, nls.vals, nls.Jv, nls.Jtv)
```

Comment:
After checking back the definition of ``jac_op_residual`` from NLPModels [here](https://github.com/JuliaSmoothOptimizers/NLPModels.jl/blob/main/src/nls/api.jl#L167), I feel like I should simply remove this ``jac_op_residual`` overwrite because I can simply call ``jac_coord_residual!`` and ``jac_op_residual!`` separately instead.